bee-hive-demos/demos/summary.ai/agent_store.json

```
@@ -0,0 +1 @@
+{"Search Arxiv": "asst_67a514988a306469596fffa6", "Intermediary": "asst_67a514988a306469596fffa8", "Filter": "asst_67a514988a306469596fffaa", "Summary": "asst_67a514988a306469596fffac"}
```

you might want to commit this as an empty dict and then later (after this PR merges) add it to the gitignore so new changes don't keep getting pushed

```suggestion
{}
```

---

Good point here, I think this also happens with the .env file if but I'll double check the gitignore